### DISCUSSION TITLE
2023-05-17 Weekly Meeting 107 Notes/Agenda

### DISCUSSION THREAD

**Discussion by funksoup** [omigroup/omigroup#390] [General]
Weekly Meeting 107 is happening on Wednesday, May 17, 2023 from 9:00 AM PST (16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

UPDATES:

MSF-delegates:

- There is a board election coming up - it's important that we/OMI have a representative on the MSF board, so we can continue to be a voice for independent creators, small orgs, open source and community/grassroots orgs. Elections will be in a few weeks but we can start preparing our community for that now. (There are some documents we will need to fill out).

- Interoperability issue to bring to the attention of MSF and industry in general (via @aaronfranke):
-- There are competing standards between OMI's [OMI_collider](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_collider) and [OMI_physics_body](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_physics_body) glTF extensions and Microsoft's physics https://github.com/KhronosGroup/glTF/pull/2257.
-- Evo will interview @aaronfranke and then post the video so that we can get more engagement on this issue.
-- How can we get more engagement? Email the MSF general list (but make the email shorter - make ask clear)
-- Evo suggested Ben Houston at ThreeKit - maybe hit up specific people like this on LinkedIn?

-- Issue is described in detail in this [OMI_collider and OMI_physics_body pull request](https://github.com/KhronosGroup/glTF/pull/2258) and there's a bulleted list of specific differences and impacts. 

Media group: 

GLTF group:

WORLD HOPS:
Status of metatraversal world hop? Real world/virtual world hybrid event for [AWE](https://www.awexr.com/)
AWE conference is May 31-June 2 in Santa Clara, CA. If we can get some folks who will be onsite at AWE to help facilitate, we could do an OMI metatraversal world hop, similar to what we did at the OMI 100th meeting anniversary event. 

We could visit / meet in these spaces (via @madjin): 
https://hyperfy.io/msf
https://hyperfy.io/neonstreet
https://hyperfy.io/omi
https://framevr.io/omi

-- Keyframe will also be planning an event on May 24 - 'Workshopping the Metaverse' in Spatial. Will be an open format -- people invited to come and ask questions. Do we have OMI slides for him to use? (Check with @mrmetaverse).

- We will need folks to be welcomers / hosts that can facilitate people joining the world hop to meet & greet.

To follow up on organizing the proposed world hop for AWE, we can use our [pro temp events channel in Discord](https://discordapp.com/channels/770382203782692945/1052644259741843526).

**Comment by mrmetaverse** [comment]
- Voxels airdropped me @mrmetaverse a extra parcel and I want to open up the build to the OMI community 
- [untitled,xyz](https://twitter.com/untitledxyz_) launched ARQs which are interoperable lego pieces of sorts to assemble metaverse spaces in a modular fashion. 
- Evo is also interested in metaverse architecture ("high end") and would love to see your designs. 
- Want to add your interoperable space to our repo for experimentation? Upload it to our [assets](https://github.com/omigroup/omigroup/tree/main/assets) folder <3.

**Comment by mrmetaverse** [comment]
## CoXist 

coXist - immersive avatar roleplaying games geared toward increasing cultural awareness and social tolerance - running it with some Ethiopian folks to build awareness between cultures in different regions (Tigrayan, Oromaic, Amharic): https://coda.io/@funksoup/immersive-learning/co-x-ist-20

**Comment by mrmetaverse** [comment]
## mapping the metaverse 
- Keyframe joined us again today! Mapping is picking back up, and we want your participation OMI. 
- Maybe a working group, maybe an effort under the experimental group in partnership with the media team, etc. 
  - Adding to OMI/experimental for discussion. 
- seeking help with the open metaverse part of the map, by our standards. Define and collaborate as a group. 
- My mapping efforts: https://keyframe-entertainment.com/miw/
- WebXR specific: https://github.com/KEYFRAME-SEHU/mapping/blob/main/WebXR_platforms.md

**Comment by mrmetaverse** [comment]
## MSF Delegates 
- re: proposed OMI/MSF collab for the world hop for the 19th. Ben Erwin is pressing onward with or without OMI. If OMI can't provide support we may want to back out. Sooner than later, it depends on if anyone can participate on behalf of OMI. 
  - Ben is pulling together a itinerary. 
  - Evo has no time to plan, so it has to be manageable and small. Maybe 2-3 worlds in each section. 
  - Would another time work better? Not likely for many of our members. 
  - Ben is giving a talk on May 31st at AWE on MetaTraversal, so including us is preferred. 
  - Jin may be able to squeeze something in. 
  - Most people have trouble helping with the planning. Jin is happy to help and host part of the event but many of us cannot help with planning due to conflicting deliverables and priorities. 
- Jin built a space in hyperfy for the MSF wearables interop group. 
  - it is a lovely space to look at the meeting notes in a more immersive experience 
  - https://hyperfy.io/msf
  - 
<img width="639" alt="image" src="https://github.com/omigroup/omigroup/assets/63426722/65613020-c7cc-4ad3-8213-529ad29cae94">
- 
<img width="608" alt="image" src="https://github.com/omigroup/omigroup/assets/63426722/3f7fc530-152d-4e84-844b-89f81ba40b9a">

**Comment by mrmetaverse** [comment]
## Going to AWE? 
Neil Trevitt asked for an update on portal interop during 20 minutes at 12:30 on May 31st, in room 203 at AWS, in the Metaverse Standards Forum room.

**Reply by mrmetaverse** [reply]
# 501c6 umbrella -> 501c3 umbrella 

Ready to be submitted. Waiting for @mrmetaverse to add missing info, and then submit it!

**Reply by mrmetaverse** [reply]
By Friday!

**Comment by mrmetaverse** [comment]
## Media 

- unblocked, lets get them some content to pub!

**Comment by mrmetaverse** [comment]
# Portal Crawl this Friday! 

Wear your cutest avatars! #389 
Need a calendar invite? email [metatraversal@gmail.com](mailto:metatraversal@gmail.com)

**Comment by mrmetaverse** [comment]
# glTF group 
- iteraeting on the physics joint 
- working on a validator and need code review 
- robert is working on physics material 
- we are working on merging our efforts to find common ground with microsofts approach. there is a MSF meeting today on physics. We will attend to ask questions and promote our approach to it as well. Our approach is meant to be more interoperable, and we want to find common ground. 
- KHR Audio continues: 2 weeks ago we tried to implement KHR Audio inside of godot engine but it is impossible because parts of the spec disagree with each other and lack consistency. Aaron raised abotu 10 issues and we will work to resolve the inconsistencies. 
- we have been shouting at the top of our lungs (metaphorically) and haven't gotten much engagement. People don't seem to actually care enough to take action on this. 
- Evo to interview Aaron about the physics issues. We are prioritizing this for next week. 🚨 challenge: Get it done before next weeks meeting <3

**Reply by mrmetaverse** [reply]
we want to get more people involved. Lets get the word out.

**Reply by mrmetaverse** [reply]
need more people reviewing overall. Its a problem with many orgs pushing standards.

**Comment by mrmetaverse** [comment]
# we need to be the change we want to see in the World.

**Comment by mrmetaverse** [comment]
Second life is going all in on glTF and getting everything to meet the spec. 
Can we contact SL to see if they would be open to using extensions like OMI_seat, and audio? 

Another platform decided to adopt glTF this week, and its Roblox.

**Comment by mrmetaverse** [comment]
# Once we have grants 
- lets fund efforts like this: WebXR doesn't work on linux. Nova has a solution. What would it take to put together a grant for this spec? "Make WebXR work on linux". 
- are there other efforts like this we should consider sponsoring and evangelizing?

**Reply by mrmetaverse** [reply]
frankly if we could put a little money in the pockets of metaverse builders by subsidizing their open metaverse efforts, I think that is a great thing overall.
